css/index.css

```diff
@@ -104,14 +109,44 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+.boxes div:nth-child(1) {
```

Comment:
HTML elements are capable of having multiple class names and ids. Your method works fine, but an id to each box will allow less typing and accessibility to those elements.

Reply:
Oh, thank you!
I have had trouble with this in the past. This was the best way I could find that didn't mess up the whole page. I did not know I could have multiple classes, that would have made things easier.